spec/controllers/users/passwords_controller_spec.rb

```diff
@@ -101,13 +101,15 @@
         stub_analytics
         allow(@analytics).to receive(:track_event)
 
-        params = { password: 'password', reset_password_token: 'foo' }
+        password = 'a really long passw0rd'
 
-        user = instance_double('User', uuid: '123')
+        params = { password: password, reset_password_token: 'foo' }
+
+        user = User.new(uuid: '123')
```

Comment:
not sure. I was just trying to preserve what was already there, using a real User object so that the `.valid?` method worked as expected.